src/frontend/efiling-frontend/src/domain/package/package-confirmation/PackageConfirmation.js

```diff
@@ -51,6 +52,7 @@ export default function PackageConfirmation({
   const [submissionFee, setSubmissionFee] = useState(null);
   const [showPayment, setShowPayment] = useState(false);
   const [showUpload, setShowUpload] = useState(false);
+  const [show, setShow] = useState(false);
```

Comment:
Maybe rename "show" variable to "showToast".   Helps clarify the code if we're showing more than one thing.

Reply:
good call just made the change